yarn-project/simulator/src/public/fixtures/index.ts

```diff
@@ -167,6 +167,7 @@ export function createTxForPublicCalls(
   appExecutionRequests: PublicExecutionRequest[],
   teardownExecutionRequest?: PublicExecutionRequest,
   gasUsedByPrivate: Gas = Gas.empty(),
+  firstNullifier: Fr = Fr.random(),
```

Comment:
Shouldn't this parameter always be provided? I fear the default value could cause damage in a refactor